src/ring_theory/ideal/basic.lean

```diff
@@ -237,20 +238,17 @@ end⟩
 instance is_maximal.is_prime' (I : ideal α) : ∀ [H : I.is_maximal], I.is_prime :=
 is_maximal.is_prime
 
+instance coatomic_lattice : is_coatomic (ideal α) :=
```

Comment:
Does this work?
```suggestion
instance : is_coatomic (ideal α) :=
```
If this doesn't complain at you, I think this is better, because it'll give you the most canonical name. (Probably `ideal.is_coatomic`)